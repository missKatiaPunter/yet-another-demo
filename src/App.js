import './App.css';
import Counter from './components/Counter';

const App = () => {
  return (
    <div className="App">
      <h1>Hello world</h1>
      <h2>Hello Yulia</h2>
      <Counter />
    </div>
  );
}

export default App;
