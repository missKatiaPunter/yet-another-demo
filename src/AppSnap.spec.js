import React from 'react';

import renderer from 'react-test-renderer';

import App from './App';

it('checks that DOM is consistent', () => {
    const tree = renderer.create(<App />).toJSON();
    expect(tree).toMatchSnapshot();
});