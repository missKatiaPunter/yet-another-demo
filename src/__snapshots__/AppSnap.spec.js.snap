// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`checks that DOM is consistent 1`] = `
<div
  className="App"
>
  <h1>
    Hello world
  </h1>
  <h2>
    Hello Yulia
  </h2>
  <button
    className="button"
    data-testid="counter"
    onClick={[Function]}
  >
    0
  </button>
</div>
`;
